js/main.js

```diff
@@ -186,13 +222,51 @@ $form.addEventListener('submit', function (event) {
       data.quoteData.anime = randomSelectedQuote.anime;
       data.quoteData.character = randomSelectedQuote.character;
       data.quoteData.quote = randomSelectedQuote.quote;
-    });
-    xhr.send();
-  }
-  $input.value = '';
+      $viewTyping.classList.toggle('hidden');
+      $loader.classList.toggle('hidden');
+    }
+  });
+  xhr.send();
+  $secondInput.value = '';
+});
+
+$firstForm.addEventListener('submit', function (event) {
+  $loader.classList.toggle('hidden');
+  $error.classList.toggle('hidden');
+  event.preventDefault();
+  clearPage();
+  var xhr = new XMLHttpRequest();
+  var selectedAnime = $firstInput.value;
+  xhr.open('GET', `https://animechan.vercel.app/api/quotes/anime?title=${encodeURIComponent(selectedAnime)}`);
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function (event) {
+    if (xhr.response.error === 'No related quotes found!') {
+      $loader.classList.toggle('hidden');
+      $error.classList.toggle('hidden');
```

Comment:
You have this in the `if` and `else` block in a couple places. If the code appears in both the `if` and `else` block, you can just do that action outside the conditional